src/Settlements.ts

```diff
@@ -75,4 +100,18 @@ export class Settlements {
    * Create a Venue under the ownership of the Current Identity
    */
   public createVenue: ProcedureMethod<CreateVenueParams, Venue>;
+
+  /**
+   * Create an Instruction to exchange Assets
+   */
+  public addInstruction: ProcedureMethod<
+    AddInstructionWithVenueIdParams,
+    Instruction[],
+    Instruction
+  >;
+
+  /**
+   * Affirm an instruction (authorize)
+   */
+  public affirmInstruction: ProcedureMethod<BigNumber, Instruction>;
```

Comment:
This one should have `args: { id: BigNumber }` as parameters (consistency with all other methods)